index.js

```diff
@@ -57,7 +60,7 @@ module.exports = function (options, logger) {
 					cb();
 				}
 			})
-			.create();
+			.create({}, options);
```

Comment:
Correct, this provides the config-path to stylint.
`rules` overrides everything regarding linting-rules. (`setConfig` internally in stylint short-circuits if provided config during initialization)